pyomo/contrib/trustregion/PyomoInterface.py

```diff
@@ -27,13 +30,22 @@ def __init__(self, trf_block, efSet):
         self.efSet = efSet
 
     def visit(self, node, values):
+        
+        sys.stdout.write('visit: \n  node = %s\n' % node)
+        for i in range(len(values)):
+            sys.stdout.write('  values[%d] = %s\n' % (i,values[i]))
```

Comment:
This looks like debugging?  If it is, it should be removed before merging.  If it is intentional output, it should come out through a `logger`